tests/Unit/AcceleratorViewCopy/avstress_0x18.cpp

```diff
@@ -68,25 +68,33 @@ int main(){
         }
     }
 
-    if (testsToRun & 0x8) {
+// Create a vector of hc::completion_future for async copy synconization
+    std::vector<hc::completion_future> cfs;
+    
+    if (testsToRun & 0x18) {
         for(uint32_t i=0;i<testIters;i++){
             if ((i%1000 == 0)) {
                 printf ("info: running Test4 %5d/%5d\n", i, testIters);
             }
-            Test4(av);
+            cfs.push_back(Test4(av));
         }
+        for(uint32_t i=0;i<testIters;i++)
+	    cfs[i].wait();
     }
 
-    if (testsToRun & 0x10) {
+    cfs.clear(); 
+    if (testsToRun & 0xFF) {
```

Comment:
When we choose to use a mask bits, usually we'd want to be able to enable certain capabilities individually (i.e. run test 1 and test 5 only).  In this unit test, the selection of the conditions doesn't allow that.  For example, if you want to enable test5, then it's mandatory to run test1 through test5 (if I understand this test, running just test5 should be fine).